scripts/zypp-plugin.py

```diff
@@ -153,7 +159,7 @@ def PLUGINBEGIN(self, headers, body):
 
         logging.debug("headers: %s" % headers)
 
-        self.description = "zypp(%s)" % basename(readlink("/proc/%d/exe" % getppid()))
+        self.description = "zypp(%s) %s" % (basename(readlink("/proc/%d/exe" % getppid())), self.zypper_arguments())
```

Comment:
AFIAS the description has a trailing whitespace if it's not from zypper. That's not nice.